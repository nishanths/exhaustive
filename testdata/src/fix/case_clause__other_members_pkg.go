package fix

import (
	barpkg "general/y"
)

func _d() {
	var phy barpkg.Phylum
	switch phy { // want "^missing cases in switch of type bar.Phylum: Chordata, Echinodermata$"
	case barpkg.Mollusca:
		{
		}
	}
}
