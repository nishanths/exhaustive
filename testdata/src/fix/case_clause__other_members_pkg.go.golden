package fix

import (
	"fmt"
	barpkg "general/y"
)

func _d() {
	var phy barpkg.Phylum
	switch phy { // want "^missing cases in switch of type bar.Phylum: Chordata, Echinodermata$"
	case barpkg.Mollusca:
		{
		}
	case barpkg.Chordata, barpkg.Echinodermata:
		panic(fmt.Sprintf("unhandled value: %v", phy))
	}
}
